static/js/tools/map/bq_query_utils.test.ts

```diff
@@ -35,9 +35,12 @@ const TEST_PLACE: PlaceInfo = {
   selectedPlace: { dcid: "geoId/06085", name: "Santa Clara", types: [] },
 };
 const TEST_DATE = "2015";
-const TEST_METADATA: StatMetadata = {
-  measurementMethod: "testMMethod",
-  observationPeriod: "P1Y",
+const TEST_METADATA_WITH_IMPORT_NAME: StatMetadata = {
```

Comment:
Excellent.  This forms the basis for Mixer golden like test-cases: https://github.com/datacommonsorg/website/issues/1595